downstream/assemblies/aap-on-azure/assembly-aap-azure-connecting-to-aap.adoc

```diff
@@ -0,0 +1,20 @@
+ifdef::context[:parent-context: {context}]
+
+[id="aap-azure-connecting-to-aap"]
+= Connecting to {PlatformName}
+
+:context: aap-azure-connecting-to-aap
+
+[role="_abstract"]
+Once network peering is performed and your Azure routing configuration is established, you can choose how your team will access {PlatformNameShort} through your Azure network configuration.
```

Comment:
When network peering is performed and your Azure routing configuration is established, you can choose how your team accesses {PlatformNameShort} through your Azure network configuration.